jquery.flot.absRelTime.js

```diff
@@ -248,26 +248,30 @@ API.txt for details.
         [1, "year"]]);
 
     function updateAxisFirstData(plot, axis) {
-        var plotData = plot.getData(),
-            i, firstPlotData, minFirstPlotData, datapoints = plotData[0].datapoints;
-
-        if (datapoints.points.length !== 0) {
-            if (plotData[0].xaxis === axis || plotData[0].yaxis === axis) {
-                minFirstPlotData = axis.direction === "x" ? datapoints.points[0] : datapoints.points[1];
-            } else { minFirstPlotData = axis.max; }
-        } else { minFirstPlotData = axis.min; }
-
-        for (i = 1; i < plotData.length; i++) {
-            datapoints = plotData[i].datapoints;
-            if (plotData[i].xaxis === axis || plotData[i].yaxis === axis) {
-                firstPlotData = axis.direction === "x" ? datapoints.points[0] : datapoints.points[1];
-                if (minFirstPlotData > firstPlotData) {
-                    minFirstPlotData = firstPlotData;
+        if (plot.getData().length === 0) {
+            axis.relativeFirstData = 0;
```

Comment:
that looks wrong to me. The relativeFirstData should remain undefined until data is available